src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -1,86 +1,92 @@
-  @Prop({ mutable: true }) public valueAsNumber?: number;
+  @property({ attribute: 'value-as-number' }) public valueAsNumber?: number;
 
   /** Name of the inner HTMLInputElement. */
-  @Prop({ reflect: true }) public name?: string = '';
+  @property({ reflect: true }) public name?: string = '';
 
   /** The <form> element to associate the inner HTMLInputElement with. */
-  @Prop() public form?: string;
+  @property() public form?: string;
 
   /** Minimum acceptable value for the inner HTMLInputElement. */
-  @Prop() public min?: string = '0';
+  @property() public min?: string = '0';
 
   /** Maximum acceptable value for the inner HTMLInputElement. */
-  @Prop() public max?: string = '100';
+  @property() public max?: string = '100';
 
   /**
    * Readonly state for the inner HTMLInputElement.
    * Since the input range does not allow this attribute, it will be merged with the `disabled` one.
    */
-  @Prop() public readonly?: boolean = false;
+  @property({ type: Boolean }) public readonly?: boolean = false;
 
   /** Disabled state for the inner HTMLInputElement. */
-  @Prop() public disabled?: boolean = false;
+  @property({ type: Boolean }) public disabled?: boolean = false;
 
   /** Name of the icon at component's start, which will be forward to the nested `sbb-icon`. */
-  @Prop() public startIcon?: string;
+  @property({ attribute: 'start-icon' }) public startIcon?: string;
 
   /** Name of the icon at component's end, which will be forward to the nested `sbb-icon`. */
-  @Prop() public endIcon?: string;
+  @property({ attribute: 'end-icon' }) public endIcon?: string;
 
   /**
    * The ratio between the absolute value and the validity interval.
    * E.g. given `min=0`, `max=100` and `value=50`, then `_valueFraction=0.5`
    */
-  @State() private _valueFraction = 0;
+  @state() private _valueFraction = 0;
 
   /**
    * @deprecated only used for React. Will probably be removed once React 19 is available.
    */
-  @Event({ bubbles: true, cancelable: true }) public didChange: EventEmitter;
-
-  /** Host element */
-  @Element() private _element!: HTMLElement;
+  private _didChange: EventEmitter = new EventEmitter(this, events.didChange, {
+    bubbles: true,
+    cancelable: true,
+  });
 
   /** Reference to the inner HTMLInputElement with type='range'. */
   private _rangeInput!: HTMLInputElement;
 
-  public connectedCallback(): void {
+  private _abort = new ConnectedAbortController(this);
+
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    const signal = this._abort.signal;
+    this.addEventListener('keydown', (e) => this._handleKeydown(e), { signal });
     this._handleChange();
   }
 
-  @Watch('value')
-  @Watch('valueAsNumber')
-  private _syncValues(newValue: string | number = this._rangeInput?.valueAsNumber): void {
+  public override willUpdate(changedProperties: PropertyValues<this>): void {
+    if (changedProperties.has('value') && !!changedProperties.get('value')) {
```

Comment:
maybe I'm misunderstanding, but what if you like to set to 0 ?

Reply:
Changed to changedProperties.get('<prop>') !== undefined